tox.ini

```diff
@@ -25,9 +29,10 @@ deps =
     pytest44: pytest~=4.4.0
     pytest43: pytest~=4.3.0
 
-    coverage: coverage
+    coverage: pytest-cov
     xdist: pytest-xdist
```

Comment:
Hmm pytest-xdist is also looking in the source files and not the installed package

Reply:
https://github.com/pytest-dev/pytest-xdist/issues/421

Reply:
```suggestion
    xdist: pytest-xdist>=2.3.0
```